metaflow/cli.py

```diff
@@ -1060,41 +1063,25 @@ def _reconstruct_cli(params):
                     yield str(value)
 
 
-def _check(graph, flow, environment, pylint=True, warnings=False, **kwargs):
-    echo("Validating your flow...", fg="magenta", bold=False)
-    linter = lint.linter
+def _check(graph, flow, environment, is_linter=True, warnings=False, **kwargs):
+    echo(f'Validating your flow ...', fg="magenta", bold=False)
+    linter = metaflow.linters.lint.linter  # killing ambiguity
     # TODO set linter settings
     linter.run_checks(graph, **kwargs)
     echo("The graph looks good!", fg="green", bold=True, indent=True)
-    if pylint:
-        echo("Running pylint...", fg="magenta", bold=False)
+    if is_linter:  # instantiate Linter/Wrapper from Factory, run its 'lint' method
+        linter_name = FLOW_VALIDATORS["DEFAULT"]
+        echo(f'Linting with {linter_name}...', fg="magenta", bold=False)
         fname = inspect.getfile(flow.__class__)
-        pylint = PyLint(fname)
-        if pylint.has_pylint():
-            pylint_is_happy, pylint_exception_msg = pylint.run(
-                warnings=warnings,
-                pylint_config=environment.pylint_config(),
-                logger=echo_always,
-            )
-
-            if pylint_is_happy:
-                echo("Pylint is happy!", fg="green", bold=True, indent=True)
-            else:
-                echo(
-                    "Pylint couldn't analyze your code.\n\tPylint exception: %s"
-                    % pylint_exception_msg,
-                    fg="red",
-                    bold=True,
-                    indent=True,
-                )
-                echo("Skipping Pylint checks.", fg="red", bold=True, indent=True)
-        else:
-            echo(
-                "Pylint not found, so extra checks are disabled.",
-                fg="green",
-                indent=True,
-                bold=False,
-            )
+        linter_choice = LinterType.from_string(linter_name.lower())
+        LinterFactory.get_linter(linter_choice).lint(
+            fname,
+            warnings=warnings,
+            pylint_config=environment.pylint_config(),
```

Comment:
@tuulos this shall go away too, to become `lint_config` basis the default/chosen linter option (i.e `pylint` or `ruff_config`)